ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -252,6 +309,75 @@ export default class ConfirmApproveContent extends Component {
     );
   }
 
+  renderFullDetails() {
+    const { t } = this.context;
+    const {
+      assetStandard,
+      showEditApprovalPermissionModal,
+      customTokenAmount,
+      tokenAmount,
+      decimals,
+      origin,
+      setCustomAmount,
+      tokenSymbol,
+      tokenBalance,
+    } = this.props;
+    if (assetStandard === ERC20) {
+      return (
+        <div className="confirm-approve-content__full-tx-content">
+          <div className="confirm-approve-content__permission">
```

Comment:
Will be nice if this method `renderApproveContentCard` can be converted to a component itself.

Reply:
Sure we could. These components are probably going to be redesigned and rebuilt pretty soon though so it may not be worth too much effort in terms of making these more robust and composable in the short term.